backend/src/main/java/de/neuefische/teddyscoronadiaries/db/RecipeMongoDb.java

```diff
@@ -0,0 +1,11 @@
+package de.neuefische.teddyscoronadiaries.db;
+
+import de.neuefische.teddyscoronadiaries.model.recipe.Recipe;
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+import java.util.List;
+
+public interface RecipeMongoDb extends PagingAndSortingRepository<Recipe, String> {
```

Comment:
Maybe create a seperate Recipe-model to use for saving in the DB?
This would allow it to differ from your backend-Recipe (e.g. add a UserID if each recipe should be assigned to a different User)

Reply:
will keep this in mind 👍🏻